lib/response.js

```diff
@@ -478,16 +478,19 @@ module.exports = {
 
   /**
    * Checks if the request is writable.
-   * Tests for the existence of the socket
-   * as node sometimes does not set it.
    *
    * @return {Boolean}
    * @api private
    */
 
   get writable() {
+    // can't write any more after response finished
```

Comment:
haha, I think support by node core is more reasonable.

https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js#L61 `response.writable` is never changed.